### PR TITLE
Elastic scale patch

### PR DESCRIPTION
**Pull request by PhilBastian**
fix results count not showing and enable deletion of non-tracked endpoints

### CODE REVIEW

File: src/Frontend/src/components/heartbeats/HealthyEndpoints.vue [hunk 1]
```diff
@@ -3,6 +3,7 @@ import NoData from "../NoData.vue";
 import { ColumnNames, useHeartbeatsStore } from "@/stores/HeartbeatsStore";
 import { storeToRefs } from "pinia";
 import HeartbeatsList from "./HeartbeatsList.vue";
+import ResultsCount from "../ResultsCount.vue";
```
Comment (johnsimons) [inline src/Frontend/src/components/heartbeats/HealthyEndpoints.vue:6]: This is really weird, how come this did not break the build?

File: src/Frontend/src/components/heartbeats/EndpointInstances.vue [hunk 2]
```diff
@@ -84,6 +87,16 @@ async function deleteInstance(instance: EndpointsView) {
   }
 }
 
+async function deleteAllInstances() {
+  try {
+    await Promise.all(sortedInstances.value.filter((instance) => instance.name === endpointName).map((instance) => deleteInstance(instance)));
+    useShowToast(TYPE.SUCCESS, "Endpoint instance deleted", "", false, { timeout: 1000 });
```
Comment (johnsimons) [inline src/Frontend/src/components/heartbeats/EndpointInstances.vue:93]: Should this say
```suggestion
    useShowToast(TYPE.SUCCESS, "Endpoint deleted", "", false, { timeout: 1000 });
```

**Review by johnsimons — CHANGES_REQUESTED** [review]

**Review by johnsimons — CHANGES_REQUESTED** [review]
I have tested this and a few things that don't look right.
- I think the "delete button" should have a warning dialog given it is a dangerous action, thoughts?
- When the deletion occurs, we get multiple toasters, I assume one per instance, to me it would make more sense to be a single toaster

**Comment by PhilBastian** [comment]
> I have tested this and a few things that don't look right.
> 
> * I think the "delete button" should have a warning dialog given it is a dangerous action, thoughts?
> * When the deletion occurs, we get multiple toasters, I assume one per instance, to me it would make more sense to be a single toaster

the other delete buttons don't give a warning
there should only be one toast message. I wonder if it's something to do with the route change?

**Comment by johnsimons** [comment]
> the other delete buttons don't give a warning

the others are single this one is bulk delete

**Review by johnsimons — APPROVED** [review]
I don't think we need the warning dialog for delete endpoint.

**Pull request merged**